src/tomato/drivers/biologic/main.py

```diff
@@ -21,7 +99,7 @@ def get_status(
     dllpath: str = None,
     lockpath: str = None,
     **kwargs: dict,
-) -> tuple[float, dict]:
+    ) -> tuple[float, dict]:
```

Comment:
Please reformat the code using `black`. This was fine as it was...

Reply:
Ok, I will reformat using black.